### DISCUSSION TITLE
rule.name is too long to escalate an alert into a case in the so_case.title field(v2.3.190)

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#9779] [Unsupported Versions] [answered]
Hi I've looked through the other posts and can't see this covered.

We are running a distributed deployment on SO v2.3.190 and have come across an issue regarding escalating an alert into a case if the **rule.name** is very long (too long for the **so_case.title field**). 
Today I received a 400 error when escalating an alert "ET ATTACK_RESPONSE Possible Lateral Movement - File Creation Request in Remote System32 Directory (T1105)" into a case, and when I tried manually creating a case with the same name the error occurred, but with a shorter name it worked.

I wondered if anyone knows of a workaround (apart from manually creating cases and adding events)?

Many thanks.

![image](https://user-images.githubusercontent.com/61547463/218525796-664f2ee4-99d8-4fa5-8cc5-472240299eda.png)

**Comment by dougburks** [comment, marked as answer]
I've confirmed this issue and it will be resolved in the upcoming 2.3.220 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/9781

**Reply by cyberkimh** [reply]
Thanks so much!
